doc/man1/dgst.pod

```diff
@@ -8,7 +8,7 @@ dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blak
 
 B<openssl> B<dgst>
 [B<-help>]
-[B<-I<digest>>]
+[B<-[digest]>]
```

Comment:
The problem is that the digests/encryption that are available depends on the build options.  So I tried to say "any available" most of the time.  But I know it's a side note :)